test/e2e/feature/feature.go

```diff
@@ -307,11 +307,6 @@ var (
 	// TODO: remove when SELinuxMount feature gate is enabled by default.
 	SELinuxMountReadWriteOncePodOnly = framework.WithFeature(framework.ValidFeatures.Add("SELinuxMountReadWriteOncePodOnly"))
 
-	// Owner: sig-network
-	// Marks tests of KEP-1880 that require the `MultiCIDRServiceAllocator` feature gate
-	// and the networking.k8s.io/v1alpha1 API.
-	ServiceCIDRs = framework.WithFeature(framework.ValidFeatures.Add("ServiceCIDRs"))
-
```

Comment:
At the moment, we still need one such `Feature` for each `FeatureGate`. 

We need it because there might be jobs which have not been converted to use `--label-filter`. Such jobs only skip if a test has a `Feature:<something>` tag.

Reply:
I see, this is exactly what is happening